enclave_manager/avalon_enclave_manager/sgx_work_order_request.py

```diff
@@ -34,8 +34,10 @@ def execute(self):
         encrypted_request = crypto.byte_array_to_base64(serialized_byte_array)
 
         try:
-            encoded_encrypted_response = \
-                enclave.HandleWorkOrderRequest(encrypted_request)
+            # Work order extended data is not used in Singleton,
+            # hence passing empty value
+            encoded_encrypted_response = enclave.HandleWorkOrderRequest(
+                encrypted_request, "", enclave.SINGLETON_ENCLAVE)
```

Comment:
Is this file specific to singleton work order handling ? I would suggest to add enclave type in SgxWorkOrderRequest constructor.  By default the value can be SINGLETON_ENCLAVE if it's not initialized.

Reply:
Ok